src/App.test.jsx

```diff
@@ -0,0 +1,62 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
+import * as mock from './utils/mockData';
+
+import App from './App';
+
+const server = setupServer(
+  rest.get('https://pokeapi.co/api/v2/pokedex', (req, res, ctx) => {
+    return res(ctx.json(mock.pokedexes));
+  }),
+  rest.get('https://pokeapi.co/api/v2/region', (req, res, ctx) => {
+    return res(ctx.json(mock.regions));
+  }),
+  rest.get('https://pokeapi.co/api/v2/pokedex/galar', (req, res, ctx) => {
+    return res(ctx.json(mock.pokemonInRegion));
+  }),
+  rest.get('https://pokeapi.co/api/v2/pokemon/sobble', (req, res, ctx) => {
+    return res(ctx.json(mock.sobble));
+  })
+);
+
+beforeAll(() => server.listen());
+afterAll(() => server.close());
+
+test('App renders a header to the page', async () => {
```

Comment:
I might consider combining both of these tests 
https://kentcdodds.com/blog/write-fewer-longer-tests

Reply:
Great suggestion; these tests are combined in [`fa42c5d` (#3)](https://github.com/jlaurentpdx/pokemon-router-comp/pull/3/commits/fa42c5dd1f5d37c05bfd3e16573ca32108ced15c). Thank you!